pkg/rsyslog/Makefile.am.tests

```diff
@@ -0,0 +1,2549 @@
+TEST_EXTENSIONS=.sh
```

Comment:
Build wouldn't go through without adding a place holder for imemlogd here. Since I have removed all these files and came up with a diff that it applied, things will be more clear now.